.travis.yml

```diff
@@ -24,7 +23,7 @@ jobs:
       before_install:
         - sudo apt-get update
       install:
-        - pip install 'configtools<0.4.0' elasticsearch sh boto3
+        - pip install urllib3 elasticsearch sh boto3
```

Comment:
Why is urllib3 now necessary?

Reply:
Travis is throwing error without it and asked to install

Reply:
This seems weird.  Can you capture the error and report it back here?